cassandra/util.py

```diff
@@ -555,3 +555,76 @@ def _intersect(self, other):
                     if item in other:
                         isect.add(item)
             return isect
+
+from collections import Mapping
+import six
+from six.moves import cPickle
+
+class OrderedMap(Mapping):
+    '''
+    An ordered map that accepts non-hashable types for keys.
```

Comment:
Good thought. I've not added any new entities before. Is it just a matter of adding it to docs/api/index.rst and docs/api/cassandra/util.rst?